README.md

```diff
@@ -1,3 +1,4 @@
+![Python application](https://github.com/ascmitc/mhl/workflows/Python%20application/badge.svg) [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
```

Comment:
depending on your answer to my question above I assume this `Python application` would change to match as well.